types/telegram-web-app/index.d.ts

```diff
@@ -157,12 +169,15 @@ interface WebApp {
      */
     sendData(data: string): void;
     /**
-     * A method that opens a link in an external browser. The Web App will not
-     * be closed. Note that this method can be called only in response to the
-     * user interaction with the Web App interface (e.g. click inside the Web
-     * App or on the main button)
+     * A method that opens a link in an external browser.
```

Comment:
What's the point of this diff? It looks like it just reformats the comment. In that case, please revert it.